delivery_driver/models/stock_move_line.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2024 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+
+
+from odoo import fields, models
+
+
+class StockMoveLine(models.Model):
+    _inherit = "stock.move.line"
+
+    driver_id = fields.Many2one(related="picking_id.driver_id", store="True")
```

Comment:
If someone wants to choose another delivery driver, domain must be present.

```suggestion
    driver_id = fields.Many2one(related="picking_id.driver_id", store="True", domain="[('is_company', '=', False)]")
```

Reply:
👌🏻 Added